src/app/components/favorites/favorites.component.html

```diff
@@ -20,24 +20,19 @@
         selectionMode="multiple"
         [navigate]="false"
         [sorting]="['modifiedAt', 'desc']"
+        sortingMode="client"
```

Comment:
maybe easier to put inside the https://github.com/Alfresco/alfresco-content-app/blob/develop/src/app/directives/document-list.directive.ts#L40 ?

Reply:
was thinking about this, but  i didn't want to break anything relying on the  default which is  'server'

Reply:
ok, no worries